website/source/docs/platform/k8s/upgrading.html.md

```diff
@@ -22,7 +22,7 @@ be updated in batches.
 
 To initiate the upgrade, change the `server.image` value to the
 desired Consul version. For illustrative purposes, the example below will
-use `consul:123.456`. Also set the `server.updatePartition` value
+use `consul:123.456`. Also, set the `server.updatePartition` value
```

Comment:
now I'm always going to be unsure if this is your grammar skillz or grammarly's 😆